sway/input/seatop_default.c

```diff
@@ -709,8 +710,17 @@ static void handle_pointer_axis(struct sway_seat *seat,
 			struct sway_node *active =
 				seat_get_active_tiling_child(seat, tabcontainer);
 			list_t *siblings = container_get_siblings(cont);
+			int delta_discrete = event->delta_discrete;
+			if (delta_discrete == 0) {
+				scroll_distance_add(event->delta);
+				float dist = scroll_distance_get();
+				delta_discrete = round(dist / 10.);
```

Comment:
This threshold of 10 should be user configurable I guess to make up for differences in touchpad models.